packages/docs/examples/va-slider/apiConfig.ts

```diff
@@ -0,0 +1,94 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
+
+export const vaSliderOptions: ManualApiOptions = {
+  version: '1.3.0', // Not really, just for test.
+  props: {
+    color: {
+      local: true,
+    },
+    range: {
+      local: true,
+    },
+    value: {
+      local: true,
+    },
+    trackLabel: {
+      local: true,
+    },
+    trackColor: {
+      local: true,
+    },
+    labelColor: {
+      local: true,
+    },
+    trackLabelVisible: {
+      local: true,
+    },
+    min: {
+      local: true,
+    },
+    max: {
+      local: true,
+    },
+    step: {
+      local: true,
+    },
+    label: {
+      local: true,
+    },
+    invertLabel: {
+      local: true,
+    },
+    readonly: {
+      local: true,
+    },
+    pins: {
+      local: true,
+    },
+    iconPrepend: {
+      local: true,
+    },
+    iconAppend: {
+      local: true,
+    },
+    vertical: {
+      local: true,
+    },
+    showTrack: {
+      local: true,
+    },
+    disabled: {
+      local: true,
+      version: '1.3.1', // Not really, just for test.
+    },
+  },
+  events: {
+    dragStart: {
+      local: true,
+      types: 'Vue',
```

Comment:
Why Vue?

I would understand if you expose native event, but what's the point emitting instance?

Reply:
this.$emit('drag-start', this)